udata/api_fields.py

```diff
@@ -138,18 +176,15 @@ def generate_fields(**kwargs):
     def wrapper(cls):
         read_fields = {}
         write_fields = {}
-        sortables = []
+        ref_fields = {}
+        sortables = kwargs.get("additionalSorts", [])
```

Comment:
I prefer to keep a list of dict because we may want to add more customisation in the sort in the future? For exemple a `allowDesc` bool or something else?

Reply:
Fine with me!